spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/TableOperation.scala

```diff
@@ -55,7 +55,10 @@ class TableOperation[T: ClassTag](
     val inputSmall = input[Tensor[T]](smallPos)
     val inputLarge = input[Tensor[T]](3 - smallPos)
 
-    if (expandLayer == null) expandLayer = ExpandSize(inputLarge.size())
+    val largeSize = inputLarge.size()
+    // batchSize may be not same for non-training case
```

Comment:
what do you mean by `non-training`

Reply:
ok, maybe Inference is more clear. The size of last batch may not be same with previous ones for model Inference